### PR TITLE
{bio}[foss/2022a] Giotto-Suite v3.0.1, python-louvain v0.16, SpatialIDE v1.1.3, silhouetteRank v1.0.5.13, smfishHmrf v1.3.3 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by stevenvdb**
(created using `eb --new-pr`)

Note that the instructions provided at https://giottosuite.readthedocs.io/en/latest/gettingstarted.html#installation should not be followed with this installation. Specifically, the `installGiottoEnvironment()` command should not be executed since the necessary Python packages are included in the module. At the moment, the `checkGiottoEnvironment()` command will return `FALSE` since Python packages are installed in a conda environment, which is not how it is done here. If needed, we can patch the code so `checkGiottoEnvironment()` returns `TRUE` even if no conda environment is found.

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 8 out of 16 (5 easyconfigs in total)
himem06.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/478519f30a177d807ec6a213ba8e765e for a full test report.

**Comment by stevenvdb** [comment]
> Test report by @jfgrimm **FAILED** Build succeeded for 8 out of 16 (5 easyconfigs in total) himem06.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8 See https://gist.github.com/478519f30a177d807ec6a213ba8e765e for a full test report.

I don't really understand what to do about these build failures. The first one that fails is `Seaborn-0.12.1-foss-2022a.eb`, which is an easyconfig that is not part of this PR but is already in the develop branch. The failure is because the `pandas` package is not installed, but this is provided by `SciPy-bundle/2022.05-foss-2022a` which is a indirect dependency of Seaborn through `matplotlib/3.5.2-foss-2022a`. So I am not sure what I can do to make these tests to pass.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 9 out of 9 (5 easyconfigs in total)
himem06.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/fe61368b4c0e03ec7b3edff06e1e99dd for a full test report.

**Comment by jfgrimm** [comment]
> > Test report by @jfgrimm **FAILED** Build succeeded for 8 out of 16 (5 easyconfigs in total) himem06.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8 See https://gist.github.com/478519f30a177d807ec6a213ba8e765e for a full test report.
> 
> I don't really understand what to do about these build failures. The first one that fails is `Seaborn-0.12.1-foss-2022a.eb`, which is an easyconfig that is not part of this PR but is already in the develop branch. The failure is because the `pandas` package is not installed, but this is provided by `SciPy-bundle/2022.05-foss-2022a` which is a indirect dependency of Seaborn through `matplotlib/3.5.2-foss-2022a`. So I am not sure what I can do to make these tests to pass.

ah my mistake, I've been working on updating the scipy easyblock and in the process broke my 2022a SciPy-bundle module. Forgot to remove it before testing the PR

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 13 out of 13 (5 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/bd6acbed9173ff980d45716a21b538dc for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/s/smfishHmrf/smfishHmrf-1.3.3-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['2fed45be8b5a5d47657cd4c8e9023b9ea6624863fc55a12afda092c6be37ca93']
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('Seaborn', '0.12.1'),
+    ('matplotlib', '3.5.2'),
+    ('scikit-learn', '1.1.2'),
+    ('Java', '11.0.16', '', SYSTEM),
```
Comment (boegel) [inline easybuild/easyconfigs/s/smfishHmrf/smfishHmrf-1.3.3-foss-2022a.eb:19]: We should use the `Java/11` wrapper here:

```suggestion
    ('Java', '11', '', SYSTEM),

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/s/smfishHmrf/smfishHmrf-1.3.3-foss-2022a.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/s/smfishHmrf/smfishHmrf-1.3.3-foss-2022a.eb:19]: I'll apply this change, so we can get this PR merged...

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3118.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/a8e41f79de6749315dddf3ddf13db1cb for a full test report.

**Comment by boegel** [comment]
Going in, thanks @stevenvdb!

**Pull request merged**